TTS/trainer.py

```diff
@@ -640,10 +641,22 @@ def _optimize(
                 update_lr_scheduler = scale_prev <= scaler.get_scale()
                 loss_dict["amp_scaler"] = scaler.get_scale()  # for logging
         else:
+            if hasattr(self.model, "capacitron_vae") and self.model.capacitron_vae is not None:
```

Comment:
Probably not. I'll think about it tomorrow.

Reply:
> We cannot run a specific code in the trainer for a model. It needs to be done somewhere else.
> 
> What we can do is that we can add a callback to the Trainer like `run_clip_grad()` and if it is implemented in the model, we can call it.
> 
> Would that work? @Edresson what do you think?

I think that It can work. For now, I don't see any other better way.

Reply:
Hey guys, any news on this? :)

Reply:
I also add @WeberJulian to this thread as he is going to help for completing this PR with grace. 

I'm unfortunately overwhelmed by the other tasks currently :(